ckanext/switzerland/helpers/format_utils.py

```diff
@@ -95,10 +87,43 @@ def prepare_formats_for_index(resources, format_mapping):
 
 
 # all formats that need to be mapped have to be entered in the mapping.yaml
-def _map_to_valid_format(resource_format, format_mapping):
-    resource_format_lower = resource_format.lower()
-    for key, values in format_mapping.iteritems():
-        if resource_format_lower in values:
-            return key
-    else:
-        return None
+def _map_to_valid_format(format):
```

Comment:
These functions all belong to the above refactoring of ``prepare_resource_format``: these are all subfunctions that are only called by that function. Since that function is covered by tests, I felt it was save to refactor the subfunctions.